sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Release History
 
+## 1.0.0 (2019-10-23)
+**Breaking changes**
```

Comment:
can you add a Known Issues section?  Add the AZURE_CLOUD issue with a link to the bug?